src/net/tls.cc

```diff
@@ -887,8 +897,14 @@ class tls::reloadable_credentials_base {
             }
         }
         void do_callback(std::exception_ptr ep = {}) {
-            if (_cb && !_files.empty()) {
```

Comment:
This check still seems relevant. If we have no files, we have nothing to report, because we did not in fact reload anything.